frontend/javascripts/oxalis/model/sagas/mesh_saga.ts

```diff
@@ -494,30 +523,42 @@ function* refreshMeshes(): Saga<void> {
   // By that we avoid to remove cells that got annotated during reloading from the modifiedCells set.
   const currentlyModifiedCells = new Set(modifiedCells);
   modifiedCells.clear();
+
+  const additionalCoordinatesObject = yield* select((state) => state.flycam.additionalCoordinates);
+  const additionalCoordinates = getAdditionalCoordinatesAsString(additionalCoordinatesObject);
```

Comment:
I'd rename additionalCoordinatesObject to additionalCoordinates, so that it is consistent with the store property.
additionalCoordinates could be renamed to additionalCoordinatesString or additionalCoordinateKey. if you want to be concise additionalCoordKey could be an option.